src/asm.rs

```diff
@@ -691,7 +706,10 @@ impl<'gcc, 'tcx> AsmMethods<'tcx> for CodegenCx<'gcc, 'tcx> {
             && options.contains(InlineAsmOptions::ATT_SYNTAX);
 
         // Build the template string
-        let mut template_str = String::new();
+        let mut template_str = ".pushsection .text\n".to_owned();
```

Comment:
Note to self: check why the change to these directives was done in the past to make sure this change is correct.

Reply:
This change itself is just to make the code easier to follow, it still outputs the same thing.